databuilder/databuilder/models/table_column_usage.py

```diff
@@ -6,64 +6,51 @@
 )
 
 from amundsen_rds.models import RDSModel
-from amundsen_rds.models.table import TableUsage as RDSTableUsage
 
 from databuilder.models.graph_node import GraphNode
 from databuilder.models.graph_relationship import GraphRelationship
 from databuilder.models.graph_serializable import GraphSerializable
 from databuilder.models.table_metadata import TableMetadata
 from databuilder.models.table_serializable import TableSerializable
-from databuilder.models.user import User
+from databuilder.models.usage.usage import Usage
 
 
-class ColumnReader(object):
+class ColumnReader(Usage):
```

Comment:
Is there a reason why we don't have explicit `ColumnReader` and `TableReader`. This is one thing that is still confusing for me - if we don't support per-column usage then why have ColumnReader at all?

Reply:
We plan to support in the first place (even before the project is open source)... but we found that the antlar usage extractor we have have some performance issues...

Reply:
I myself can't speak to the history, but going forward there's not many options except:

- rename the current classes, which would be a larger incompatible change
- add new duplicate classes doing the same thing but named more correctly. Not sure it would get much adoption and may add confusion
- wait until we do implement per-column usage inside the existing classes. In that case having `ColumnReader` also do table usage when column is not specified does make sense. 

If and when the last option happens, everything will make sense. Until then, we either have to accept the weirdness or be willing to make a bigger incompatible change.